Adds docs update guidelines (#8203)
PostHog/posthog.com: contents/handbook/company/docs.md

---
title: Documentation
sidebar: Handbook
showTitle: true
---

## Responsibilities

- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
  - Documenting new features when they're launched.
  - Correcting mistakes reported by users.

>   - Clarifying documentation based on user feedback and support tickets.

Reviewer: Downside of this is that it makes it dramatically harder for other teams though, given that the PostHog repo has other requirements (e.g. linting, titling) than the .com repo. Also the checks, etc.

>   - Clarifying documentation based on user feedback and support tickets.

Reviewer: ```suggestion
  - Clarifying documentation based on user feedback and support tickets.
  - Ensuring public betas have documentation which is linked to from feature preview menu
```

Reviewer: This is often missed and will avoid people making excuses about 'Oh I didn't document it because it was in beta' or anything

>   - Clarifying documentation based on user feedback and support tickets.

Reviewer 2: That's not necessarily true. In good monorepo fashion, we should not run tests, checked, deploys for pure docs changes.

>   - Clarifying documentation based on user feedback and support tickets.

Author: Any thoughts on this idea @corywatilo @smallbrownbike?

> - **Content marketing** is responsible for improving the quality of documentation. This means:

Author: ```suggestion
- **Content marketing** is responsible for improving the overall quality of the docs. This means:
```

> - **Content marketing** is responsible for improving the overall quality of the docs. This means:

Author: ```suggestion
- **Content marketing** is responsible for improving the overall quality of the docs. This means:
```

>   - Polishing draft documentation created by product teams.

Author: ```suggestion
  - Reviewing draft documentation created by product teams.
```

  - Identifying and improving low quality documentation.

>    - Ensuring screenshots and other visual elements are up-to-date.

Author: ```suggestion
  - Ensuring screenshots and other visual elements are up-to-date.
```

>   - Shipping supplementary docs and tutorials based on feedback and emerging use cases.

Reviewer 3: Im a bit concerned this implies product teams can just ship rubbish in the hopes we'll clean up after them. Can we add a bullet point in the **Product teams** section that says something along the lines of "you can tag the content marketing team to review your PRs to ensure your writing is clear" (obviiously phrased nicer)

Reviewer 3: Its not clear on github but my comment was not refering to line 18 specifically, but rather lines 9-18 collectively

Author: Oh, I see.

Looking at this again, I think no wording tweak is going to eliminate laziness / bad faith drafts that pushes the work onto other people. I think, given our culture, it's reasonable to expect people to do a half decent job and, if fail to do so, this should be addressed via feedback.

>   - Shipping supplementary documentation and tutorials based on user feedback, and emerging use cases.
>   - Updating screenshots and other elements where needed.

Author: ```suggestion
   - Ensuring screenshots and other visual elements are up-to-date.
  - Shipping supplementary documentation and tutorials based on user feedback, and emerging use cases.
```

>   - Shipping supplementary documentation and tutorials based on user feedback, and emerging use cases.

Author: ```suggestion
  - Shipping supplementary docs and tutorials based on feedback and emerging use cases.
```

> - **Website & Docs** is responsible for design, organization, and discovery. This means:

Reviewer 4: I guess it's safe to rename Website & Docs to just Website if the content of the docs are out of their purview

  - The design and content of index pages.
  - The overall layout of docs and how they're organized.

>   - In-page elements and components – e.g. light and dark mode.

Author: ```suggestion
  - In-page elements and components – e.g. light and dark mode screenshots.
```

  - Website search and other elements that help users find the answers they need.

## Frequently asked questions

### I'm really busy, can the content team write docs for me?

No. The content team isn't big enough to own all documentation. It also lacks the context necessary to document new features. First drafts of documentation must always come from the relevant product team.

If you need help updating documentation:

- Write a draft that covers the basics, which the content team can then help review and polish.

> - In the event multiple pages needed updating, create an example of changes needed and then request help to complete the rest.

Author: ```suggestion
- In the event multiple docs pages need updating, create an example of changes needed and then request help to complete the rest.
```

> **Bottom line:** It's much easier for the content team to improve a draft than write completely new documentation, especially when documenting new features. PR > Issues.

Author: ```suggestion
**Bottom line:** It's much easier for the content team to improve a draft than write completely new documentation, especially when documenting new features. Pull requests > Issues.
```
